File: Mazal-Tov WebApi/DTO/matchmaker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
  public  class Matchmaker
    {
        public int Id { get; set; }
        public string Address { get; set; }
        public string MedicalProblem { get; set; }
        public int ChasidutId { get; set; }
        public int NumMatching { get; set; }
        public int ChildrenMatchmaking { get; set; }
        public int HowSend { get; set; }
        public bool IsCommitment { get; set; }
        public bool IsAgreement { get; set; }
        public int Status { get; set; }
        public int CityId { get; set; }
        public int SectorId { get; set; }
        public int WorkerId { get; set; }

    }
}

File: mazal-tov/src/app/classes/matchmaker.ts
import { User } from './user';
import { Worker } from './worker';


export class Matchmaker {
      _id: string;
      address: string;
      medicalProblem: string;
      chasidut: string;
      numMatching: number;
      childrenMatchmaking: number;
      howSend: number;
      isCommitment: boolean;
      isAgreement: boolean;
      city: string;
      sector: string;
      user: User = new User();
      tell: string;
      phone: string;
      fax: string;
      experiance:number;
      confidentComment:string;
      workerRemark:string;
      point:number;

}
